docs/ides/vscode.rst.include

```diff
@@ -10,13 +10,298 @@ It can be used together with any other coding language or be used to edit config
+       command ``import matplotlib.pyplot as plt`` you will import the exact same function as in the example before but you don't have to write as much in order to call the function: ``plt.plot(x, y)``.
+
+
+Extensions
+^^^^^^^^^^
+
+1. To install an extension click on the extensions tab in the activity bar (see below).
+
+2. Then search for the extension you want to install and click on install.
+
+   .. image:: images/Vscode-Extensions.png
+      :width: 600
+      :align: center
+    
+
+Extensions can also be installed by following `these instructions <https://code.visualstudio.com/docs/editor/extension-marketplace>`__.
+
+
+Debugging
+^^^^^^^^^
+
+Debugging is the process of finding and fixing errors within a script. 
+VSCode provides a powerful debugging tool that allows you to set breakpoints, step through code, and inspect variables.
+
+1. Set a breakpoint, click on the left of the line number where you want to pause the execution of your code. A red dot will appear to indicate that a breakpoint has been set, it is possible to set multiple breakpoints in your code. To remove a breakpoint, click on the red dot again.
+
+2. Start a debugging session, click on the *Run and Debug* tab in the sidebar and select *Run and Debug*. This will open the debug panel at the top of the screen and start the debugging session.
+
+   .. image:: images/Vscode-Debugging.png
+      :width: 600
+      :align: center
+
+The buttons in the debug panel allow you to control the execution of your code. From left to right the buttons are:
+
+**Continue:** Continue running the code until the next breakpoint is reached.
+
+**Step over:** Execute the next line of code. 
+
+**Step into:** Execute the next line of code, if the next line is a function call, step into the function.
+
+**Step out:** Execute the remaining lines of the function and return to the line where the function was called.
+
+**Restart:** Restart the debugging session.
+
+**Stop:** Stop the debugging session.
+
+3. Follow the variables in your code and their values on the left.
+
+   .. image:: images/Vscode-Debugging2.png
+      :width: 600
+      :align: center
+
+
+
+Virtual environment (venv)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Virtual environments are isolated spaces in computing to manage software dependencies separately. They prevent conflicts, ensure project portability, and maintain system stability. 
+You can read more about virtual environments `here <https://pythonsupport.dtu.dk/python/environments.html>`__.
+
+The interpreter is the program that executes the code you write. It is important to ensure that the interpreter is pointing to the correct Python executable (environment).
+To change the interpreter location open a python file and click on the Python version in the bottom left corner of the screen (see below).
+
+.. image:: images/Vscode-interpreter.png
+   :width: 600
+   :align: center
+
+
+Install requirements.txt 
+^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I guess this is also an installation thing, not really vs code specific, but some hints at opening the terminal in vs code would be ideal.